packages/react/src/Panels/WorkspacePanel.tsx

```diff
@@ -241,7 +241,7 @@ function PreviewsSection({
     <Panel
       id={hasPreviews ? 'previews-opened' : 'previews-closed'}
       defaultSize={hasPreviews ? 50 : 0}
-      minSize={10}
+      minSize={3}
```

Comment:
This `3` here is relative %-unit. On smaller screens this looks like:

![image](https://github.com/user-attachments/assets/fe64c0d2-8348-4dbe-9029-f24d97c431dd)

I think we need to calculate the `minSize` based on window height here.

Reply:
Good catch.  That makes this PR a little more complex. I pushed changes that will calculate the percentage height based on a static pixel value